### PR TITLE
Miscellaneous bugfixes.

### PR DESCRIPTION
**Pull request by phulin**

### CODE REVIEW

**Comment by phulin** [comment]
@gausie can you review

File: src/resources/2018/SongBoom.ts [hunk 1]
```diff
@@ -24,6 +24,14 @@ export const songBoomSongs = new Set<string>([
   "Total Eclipse of Your Meat",
 ]);
 
+const keywords = {
```
Comment (gausie) [inline src/resources/2018/SongBoom.ts:27]: Can we stop these three repeated sections and just say

```ts
type SongBoomSong = keyof keywords | null;
```
and replace things like `songBoomSongs.has(x)` with `x in keywords`

In doing so I would rename `keywords` to `songBoomSongs`

**Review by gausie — CHANGES_REQUESTED** [review]
Just one change

File: src/resources/2018/SongBoom.ts (hunk 1 above)
Comment (phulin) [reply src/resources/2018/SongBoom.ts:11]: The keywords don't need to be exposed. Otherwise done.

**Review by gausie — APPROVED** [review]

**Pull request merged**